.luacheckrc

```diff
@@ -0,0 +1,13 @@
+
+unused_args = false
```

Comment:
Ouh, so you shouldn't use `:`? Then I have to edit my `digiline_remote` mod fast before anyone uses it.